parlai/agents/transformer/image_polyencoder.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python3
```

Comment:
I feel like this model deserves to go in image_seq2seq or another folder. It's deserves more than "minor transformer variant mixed in with polyencoder"

Reply:
hmm i think you're onto something, especially since they both use the `ContextWithImageEncoder`.

Not sure what the right move is here - perhaps combine in one folder `multimodal/image_seq2seq` and `multimodal/image_polyencoder`, though this is a bit opaque. 

I would be ok with leaving as is unless @stephenroller feels strongly